includes/EXAMPLE.settings.local.inc.php

```diff
@@ -30,4 +30,9 @@
 	/**
 	 * The address which is used to access this system.
 	 */
-	defined('SITE_URL')			?	null	:	define('SITE_URL', 'http://localhost/');
\ No newline at end of file
+	defined('SITE_URL')			?	null	:	define('SITE_URL', 'http://localhost/');
+
+	/**
+	 * The timezone to be used by the system.
+	 */
+	defined("TZ")				?	null	:	define("TZ", "UTC");
```

Comment:
`TZ` can be unclear, a better name might be `TIMEZONE`. Can you adjust to make the constant name more descriptive?